.gitignore

```diff
@@ -5,3 +5,4 @@
 .rvmrc
 /coverage
 Gemfile.lock
+.ruby-version
```

Comment:
This was just for my personal testing.  This isn't prepared. Seeing what it takes to pass build.